opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -273,10 +291,13 @@ public SourceResponse query(QueryRequest queryRequest) throws UnsupportedQueryEx
                 Metacard metacard = null;
                 List<Result> resultQueue = new ArrayList<Result>();
                 try {
-                    metacard = inputTransformer.transform(responseStream);
-                } catch (IOException e) {
-                    LOGGER.debug("Problem with transformation.", e);
-                } catch (CatalogTransformerException e) {
+                    final byte[] bytes = IOUtils.toByteArray(responseStream);
```

Comment:
The performance will most likely be negligible. The point is, don't blow up memory. Just about any magic number will be better than simply dumping a byte stream that you do not control directly into memory.